Backend/app/rider-platform/rider-app/Main/src/Storage/Tabular/Issue.hs

```diff
@@ -34,10 +35,11 @@ mkPersist
     IssueT sql=issue
       id Text
       customerId SPerson.PersonTId
-      bookingId SQuote.QuoteTId Maybe
+      rideBookingId SQuote.QuoteTId Maybe
```

Comment:
why this changed? it should be `quoteId` at all

Reply:
> should

In local query is failing as in issue table column name is rideBookingId

Reply:
then rename db column, instead of renaming this.

Reply:
OKayy